connectors/open_policy_agent/go/api_default.go

```diff
@@ -13,12 +13,18 @@
 package openapiserver
 
 import (
+	"fmt"
+	"io/ioutil"
 	"net/http"
 
 	"github.com/gin-gonic/gin"
 )
 
 // GetPoliciesDecisions - getPoliciesDecisions
 func GetPoliciesDecisions(c *gin.Context) {
+	fmt.Println("in GetPoliciesDecisions of V2 OPA Connector!")
```

Comment:
why are we using fmt.println/printf? where does it go? should this be log?

Reply:
fixed to use log package. Please check. Thanks.